Apps/PatientDataAccess/src/PatientDataRepository.cs

```diff
@@ -28,22 +29,11 @@ namespace HealthGateway.PatientDataAccess
     /// <summary>
     /// Provides internal data access for Patient.
     /// </summary>
-    internal class PatientDataRepository : IPatientDataRepository
+    /// <param name="patientApi">The patient API to use.</param>
+    /// <param name="mapper">The injected mapper.</param>
+    [SuppressMessage("Style", "IDE0072:Switch expression should be exhaustive", Justification = "Team decision")]
```

Comment:
Moved SuppressMessage from class level to method level.  See commit: 70eccda722f1cf6b315ed776080744a92e9b6e4b